pkg/kubelet/rkt/rkt.go

```diff
@@ -82,7 +84,7 @@ const (
 // runtime implements the Containerruntime for rkt. The implementation
 // uses systemd, so in order to run this runtime, systemd must be installed
 // on the machine.
-type runtime struct {
+type Runtime struct {
```

Comment:
Sorry for missing this message last night.

This is just conflict caused by rebase. When I tried to resolve it manually, I guess I compared it to HEAD instead of 1.1 release. In HEAD, they are capitalize. I will revert this change.